html-syntax/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Syntax</title>
+  </head>
+  <body>
+    <img alt = "picture of Pikachu" title = "Pikachu" src = "https://assets.pokemon.com/assets/cms2/img/pokedex/full/025.png">
```

Comment:
You don't want spaces around your equal signs when declaring attributes, it can mess up HTML's parsing in the browser